Gemfile

```diff
@@ -16,7 +16,7 @@ gem 'figaro'
 gem 'httparty'
 gem 'identity-hostdata', git: 'https://github.com/18F/identity-hostdata.git', tag: 'v1.0.1'
 gem 'identity-logging', git: 'https://github.com/18f/identity-logging.git', tag: 'v0.1.0'
-gem 'identity_validations', git: 'https://github.com/18f/identity-validations.git', branch: 'master'
+gem 'identity_validations', git: 'https://github.com/18f/identity-validations.git', branch: 'main'
```

Comment:
bummer that this gem doesn't have any versions tagged :[

Reply:
> bummer that this gem doesn't have any versions tagged :[

Looks like the current main commit there is associated with a version bump to `0.3.2`. I could go ahead and tag that.

Reply:
That works!